display/d.mon/main.c

```diff
@@ -176,7 +178,7 @@ int main(int argc, char *argv[])
         if (list_flag->answer)
             G_warning(_("Flag -%c ignored"), list_flag->key);
         mon = G_getenv_nofatal("MONITOR");
-        if (mon) {
+        if (mon && strlen(mon) < MONITOR_NAME_LIMIT) {
```

Comment:
There is actually `GNAME_MAX`, so you that.

Reply:
Done